test/test_fx.py

```diff
@@ -757,8 +758,27 @@ def forward(self, x):
         shape_prop.ShapeProp(tc_traced).propagate(torch.rand(3, 4))
         self.assertEqual(tc_traced.graph.result.shape, ref_out.shape)
 
-
-
+    def test_find_single_partition(self):
+        class testModule(torch.nn.Module):
+            def forward(self, a, b):
+                return a + b
+        m = testModule()
+        traced = symbolic_trace(m)
+        partitioner = Partitioner()
+        devices = [{"name": "dev_0", "available_mem": float('inf')}]
+        dag = partitioner.partition_graph(traced, devices)
+        for node in traced.graph.nodes:
+            assert node.partition_ids == [1]
+        nodes = traced.graph.nodes
+        res_dag = DAG()
+        res_dag.create_node(0, [], [1], [], [])
+        res_dag.create_node(1, [0], [], [nodes[0], nodes[1]], [nodes[2]])
+        for r, d in zip(res_dag.nodes, dag.nodes):
+            assert(r.partition_id == d.partition_id)
+            assert(r.parents == d.parents)
+            assert(r.children == d.children)
+            assert(r.input_nodes == d.input_nodes)
+            assert(r.output_nodes == d.output_nodes)
```

Comment:
Hey, can we put experimental tests in another new test file? e.g. `test/test_fx_experimental.py`? We want to minimize the concurrent PRs landing adding tests to a single test file since it causes merge conflicts and delays landing for everyone

An alternative is to put new tests in a random location in this file rather than at the end so that the diff algorithm will not detect a merge conflict

Reply:
I will create a new file "test/test_fx_experimental.py". Do you think I should move the current tests in test_fx.py to "test/test_fx_experimental.py" ?